cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/navigate_to_resource_type.js

```diff
@@ -0,0 +1,261 @@
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import {
+  JOURNAL_SUBTYPES,
+  JOURNAL_FIELDS,
+  BOOK_SUBTYPES,
+  BOOK_FIELDS,
+  REPORT_SUBTYPES,
+  REPORT_FIELDS,
+  CHAPTER_SUBTYPES,
+  CHAPTER_FIELDS,
+  STUDENT_THESIS_SUBTYPES,
+  STUDENT_THESIS_FIELDS,
+  RESOURCE_TYPES,
+  RESOURCE_TYPE_FIELDS,
+} from '../../../support/data_testid_constants';
+
+const doiLink = 'https://doi.org/10.1126/science.169.3946.635';
+const filename = 'example.txt';
+
+// Feature: Creator navigates to Resource Type tab
+
+// Common steps
+Given('Creator begins registering a Registration in the Wizard with a Link', () => {
+  cy.login(USER_WITH_AUTHOR);
+  cy.startRegistrationWithLink(doiLink);
+  cy.get('[data-testid=registration-link-next-button]').should('be.enabled');
+  cy.get('[data-testid=registration-link-next-button]').click({ force: true });
+});
+Given('Creator begins registering a Registration in the Wizard with a File', () => {
+  cy.login(USER_WITH_AUTHOR);
+  cy.startRegistrationWithFile(filename);
+  cy.get('[data-testid=registration-file-start-button]').should('be.enabled');
+  cy.get('[data-testid=registration-file-start-button]').click({ force: true });
+});
+When('they navigate to the Resource Type tab', () => {
+  cy.get('[data-testid=nav-tabpanel-resource-type').click({ force: true });
+  cy.wrap(RESOURCE_TYPE_FIELDS).as('fields');
+});
+And('they click the Save button', () => {
+  cy.get('[data-testid=button-save-registration]').click({ force: true });
+  cy.get('[data-testid=button-save-registration]').should('be.enabled');

```

Comment:
rart å sjekke at den er enabled etter man har trykket?

Reply:
Det er for å sikre at lagre er gjenomført før man går videre til neste steg. Cypress trykker på ting veldig for og jeg har opplevd at lagring i dette tilfellet ikke er blitt gjennomført hvis jeg ikke venter på at knappen er enabled.